src/main/java/edu/lwtech/csd297/teachersfirst/pages/PageLoader.java

```diff
@@ -85,6 +85,7 @@ protected PageLoader(HttpServletRequest request, HttpServletResponse response) {
 		templateDataMap.put("isAdmin", isAdmin);
 		templateDataMap.put("isInstructor", isInstructor);
 		templateDataMap.put("isStudent", isStudent);
+		message = message.replace("//", "<br>\n			");
```

Comment:
The tab count there was perfect, was that just lucky?

Reply:
Yes!